#! /bin/sh /usr/share/dpatch/dpatch-run
## checkinstall symlink by Peter Eisentraut <petere@debian.org>
##
## DP: don't complain about dangling symlinks; we use those

@DPATCH@
diff -urNad egroupware~/phpgwapi/inc/functions.inc.php egroupware/phpgwapi/inc/functions.inc.php
--- egroupware~/phpgwapi/inc/functions.inc.php	2007-01-24 20:50:42.000000000 +0100
+++ egroupware/phpgwapi/inc/functions.inc.php	2006-12-29 04:12:29.000000000 +0100
@@ -28,9 +28,9 @@
	die("eGroupWare requires PHP $egw_min_php_version or greater.<br />Please contact your System Administrator to upgrade PHP!");
 }
 // check if eGW's pear repository is installed and prefer it over the other ones
-if (is_dir(EGW_SERVER_ROOT.'/egw-pear'))	
+if (is_dir(EGW_SERVER_ROOT.'/egw_pear'))	
 {
-	set_include_path(EGW_SERVER_ROOT.'/egw-pear'.PATH_SEPARATOR.get_include_path());
+	set_include_path(EGW_SERVER_ROOT.'/egw_pear'.PATH_SEPARATOR.get_include_path());
 	//echo "<p align=right>include_path='".get_include_path()."'</p>\n";
 }
 
